Assets/AWSIM/Scripts/UI/UserShortcuts.cs

```diff
@@ -0,0 +1,43 @@
+using UnityEngine;
+
+public class UserShortcuts : MonoBehaviour
+{
+    [SerializeField] private Transform egoTransform;
```

Comment:
```suggestion
    [SerializeField] private Transform egoTransform;
```

Where is this field set or initialized?

Reply:
In the inspector.